MahApps.Metro/Controls/Flyout.cs

```diff
@@ -542,6 +547,24 @@ internal void ApplyAnimation(Position position, bool animateOpacity, bool resetS
             }
         }
 
+        private void SetParentWindowEvents()
+        {
+            var window = this.TryFindParent<MetroWindow>();
+            if (window != null)
+            {
+                this.ClearParentWindowEvents(window);
+
+                PART_WindowTitle.MouseDown += window.TitleBarMouseDown;
+                PART_WindowTitle.MouseUp += window.TitleBarMouseUp;
+            }
+        }
+
+        private void ClearParentWindowEvents(MetroWindow window)
```

Comment:
this should be also called in `FlyoutsControl` to avoid memory leaks (cause it's possible to have dynamic flyouts)

```csharp
        protected override void ClearContainerForItemOverride(DependencyObject element, object item)
        {
            var window = this.TryFindParent<MetroWindow>();
            if (window != null)
            {
                ((Flyout)element).ClearParentWindowEvents(window);
            }
            base.ClearContainerForItemOverride(element, item);
        }
```